sound/soc/intel/boards/bxt_pcm512x.c

```diff
@@ -190,6 +190,12 @@ static const struct snd_soc_ops aif1_ops = {
 	.shutdown = aif1_shutdown,
 };
 
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_VIRTIO_VHOST)
+#define SSP5_NAME "vm_dai_link"
+#else
+#define SSP5_NAME "SSP5-Codec"
```

Comment:
we shouldn't even review this... The machine driver should not be one that is tied to hardware.

Reply:
@plbossart not sure what exactly you mean, but yes, this has to change, it's a part of the "hard-coded string" set, that I have to fix.

Reply:
@plbossart gone in the latest version. This file isn't modified any more now.